src/components/ChatEntry.js

```diff
@@ -1,37 +1,35 @@
-import React, { useState } from 'react';
+import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
 import TimeStamp from './TimeStamp';
 
 
 const ChatEntry = (props) => {
-  const [isLiked, setIsLiked] = useState(false);
+  const heart = props.liked ? '❤️' : '🤍';
 
   const toggleLike = () => {
-    setIsLiked(!isLiked);
-    if (!isLiked) {
-      props.increaseLikes();
-    } 
+    props.handleLikeToggle(props.id);
   };
 
-
   return (
     <div className="chat-entry local">
       <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
         <p>{props.body}</p>
         <p className="entry-time"><TimeStamp time={props.timeStamp}/></p>
-        <button className="like" onClick={toggleLike}>{isLiked ? '❤️' : '🤍'}</button>
+
+        <button className="like" onClick={toggleLike}>{heart}</button>
       </section>
     </div>
   );
 };
 
 ChatEntry.propTypes = {
+  id: PropTypes.number.isRequired,
   sender: PropTypes.string.isRequired,
   body: PropTypes.string.isRequired,
   timeStamp: PropTypes.string.isRequired,
-  liked: PropTypes.bool.isRequired
+  liked: PropTypes.bool.isRequired,
```

Comment:
Be sure to include info about the callback
```js
  handleLikeToggle: PropTypes.func,
```
We can leave it not required (since the tests won't know to pass it, resulting in a warning), though as mentioned previously, if we have any non-required props, we should make sure our code doesn't crash if they are left off, such as by only calling a function after confirming that we actual received one.